prow/cluster_lib.sh

```diff
@@ -26,7 +26,7 @@ ZONE=us-central1-f
 MACHINE_TYPE=n1-standard-4
 NUM_NODES=1
 CLUSTER_NAME=
-CLUSTER_VERSION=$(gcloud container get-server-config --project="${PROJECT_NAME}" --zone="${ZONE}" --format='value(defaultClusterVersion)')
+CLUSTER_VERSION="1.8.3-gke.0"
```

Comment:
1.8.4-gke.0 is available now. The problem with setting a fixed version is that gke does not keep them. What about something like:
IFS=';' VERSIONS=($(gcloud container get-server-config --project=istio-testing --zone=us-central1-f --format='value(validMasterVersions)'))
CLUSTER_VERSION="${VERSIONS[1]}"

Reply:
That seems works - updated.